chainer/functions/pooling/max_pooling_nd.py

```diff
@@ -257,10 +258,26 @@ def max_pooling_nd(x, ksize, stride=None, pad=0, cover_all=True):
             ``pad=p`` and ``pad=(p, p, ..., p)`` are equivalent.
         cover_all (bool): If ``True``, all spatial locations are pooled into
             some output pixels. It may make the output size larger.
+        return_indices (bool): If ``True``, pooling indices array is returned
+            together with the output variable. The returned indices are
+            expected for use by :func:`chainer.functions.upsampling_nd`.
+            Note that cuDNN will not be used for this function if
+            ``return_indices`` is set to ``True``, as cuDNN does not return
+            indices information.
 
     Returns:
-        ~chainer.Variable: Output variable.
+        ~chainer.Variable or tuple:
+            When ``return_indices`` is ``False`` (default), returns the output
+            variable. Otherwise returnes the tuple of the output variable and
```

Comment:
Could you address https://github.com/chainer/chainer/pull/4890#discussion_r195967131 here too?

Reply:
Fixed.